CoreWidgetProvider/Helpers/CPUStats.cs

```diff
@@ -103,7 +104,7 @@ internal void KillTopProcess(int cpuProcessIndex)
     public void Dispose()
     {
         procPerf.Dispose();
-        procSpeed.Dispose();
+        procPerformance.Dispose();
```

Comment:
Do we need to dispose procFrequency as well?

Reply:
definitely good point :) 
Fixed